styles.css

```diff
@@ -31,3 +37,17 @@
     top: 20px;
 }
 
+.container:hover #menu {
+    background-color: transparent;
+}
+
+.container:hover #menu::before {
```

Comment:
Nova had in haar versie de rotate op 45 graden staan waardoor het wat meer een kruis is met even verhoudingen, maar dat is meer een design keuze, want dit werkt prima.